hwdb.d/60-keyboard.hwdb

```diff
@@ -700,14 +700,21 @@ evdev:name:gpio-keys:phys:gpio-keys/input0:ev:23:dmi:*:svnHewlett-Packard:pnHPSt
 evdev:name:Huawei WMI hotkeys:dmi:bvn*:bvr*:bd*:svnHUAWEI:*
  KEYBOARD_KEY_287=f20                                   # Microphone mute button, should be micmute
 
-# Huawei MACH-WX9
+# Huawei MACH-WX9 and EUL-WX9
```

Comment:
I think the issue is, you need a comment with the name between old and new evdev: lines

Reply:
Hm, let me try. I'll ping you once I get CI to pass. Thank you.